flipt-client-java/src/main/java/io/flipt/client/models/EvalRequest.java

```diff
@@ -0,0 +1,42 @@
+package io.flipt.client.models;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.util.Map;
+
+public class EvalRequest {
```

Comment:
can we make this private or protected? I don't think we want to expose this in the public API

Reply:
@markphelps great idea!